modules/bmc_check/esbmccheck.py

```diff
@@ -929,48 +952,76 @@ def getnumbeginfuncts(_cfilepath):
 
         return listbeginnumfuct
 
-    def configureCPACheckerPath(self):
+    def configureUAPath(self):
+        return "./Ultimate.py " + self.listproperty + " 64bit "
```

Comment:
Are we using ultimate?

Reply:
Depthk used before CPAchecker and uautomizer. The cpachecker removed while we kept uautomizer tool and ultimate.py is the key for it.